tests/chart_tests/test_prometheus_blackbox_exporter.py

```diff
@@ -67,26 +77,26 @@ def test_prometheus_blackbox_exporter_deployment_custom_resources(self, kube_ver
                 },
             },
             show_only=["charts/prometheus-blackbox-exporter/templates/deployment.yaml"],
-        )[0]
-
-        assert doc["kind"] == "Deployment"
-        assert doc["metadata"]["name"] == "release-name-prometheus-blackbox-exporter"
+        )
 
+        common_blackbox_exporter_tests(docs)
+        doc = docs[0]
```

Comment:
We dont't need to assign `doc` if we're only going to use it once on the next line. Just use `docs[0]` on the next line.